### PR TITLE
Create user.go in the command package

### PR DESCRIPTION
**Pull request by NFavareto**
### Description
This PR creates a user.go in the command package. This new file has a function for obtaining loose user information using the user ID.

At the moment, user.go is tightly attached to the slack, as the command package is like that..

### How to test?
On a local or staging environment, open an incident and verify if commander's columns  is created with a value.

- commander_id
- commander_email

### Expected behavior
Hellper must be able to create the incident  without a problem and atribute a value to the commander's columns

### CODE REVIEW

File: internal/model/user.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package model
+
+
```
Comment (GLOFonseca) [inline internal/model/user.go:3]:
```suggestion
```

File: internal/bot/client.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ type Client interface {
 	CreateChannel(string) (*slack.Channel, error)
 	InviteUserToChannel(string, string) (*slack.Channel, error)
 	ListPins(string) ([]slack.Item, *slack.Paging, error)
-	GetUserInfo(string) (*slack.User, error)
+	GetUserInfoContext(context.Context, string) (*slack.User, error)
```
Comment (GLOFonseca) [inline internal/bot/client.go:15]: Great!

File: internal/commands/open.go [hunk 3]
```diff
@@ -156,14 +156,19 @@ func StartIncidentByDialog(
 		product        = submission.Product
 		commander      = submission.IncidentCommander
 		description    = submission.IncidentDescription
-
+
```
Comment (GLOFonseca) [inline internal/commands/open.go:159]:
```suggestion

```

File: internal/model/user.go [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+type User struct {
+	Id			int64       
+	SlackId		string
```
Comment (GLOFonseca) [inline internal/model/user.go:7]: Nothing critical, but this field could be called `SlackID`

```suggestion
	SlackID		string       
```

Comment (NFavareto) [reply internal/model/user.go:7]: @tiagotor  change in this files as well: 

- internal/commands/user.go

- internal/commands/user_test.go

**Pull request merged**